app/vep/utils/web_metadata.py

```diff
@@ -1,26 +1,32 @@
 from loguru import logger
 import requests
+from requests import HTTPError
 from typing import List
 from core.config import ENSEMBL_WEB_METADATA_API
 from vep.models.client_model import GenomeAnnotationProvider
+import logging
 
 async def get_genome_metadata(genome_id: str) -> GenomeAnnotationProvider:
-
     try:
-      session = requests.Session()
-      with session.get(
-          url=f"{ENSEMBL_WEB_METADATA_API}/genome/{genome_id}/details"
-      ) as response:
-          response.raise_for_status()
-          metadata = response.json()
-    except requests.exceptions.HTTPError as HTTPError:
-      logger.error(f"HTTPError: {HTTPError}")
-      return None
+        response = requests.get(
+            ENSEMBL_WEB_METADATA_API
+            + "genome/"
+            + genome_id
+            + "/dataset/genebuild/attributes?"
+            + "attribute_names=genebuild.provider_name&"
+            + "attribute_names=genebuild.display_version&"
+            + "attribute_names=genebuild.last_geneset_update"
+        )
+        response.raise_for_status()
+        attributes = {}
+        print (attributes)
+        for attribute in response.json()['attributes']:
+            name = attribute['name']
+            value = attribute['value']
+            attributes[name] = value
+        return attributes
+    except HTTPError as http_error:
+        logging.debug(http_error)
```

Comment:
Could you ask @Mehrnaz-Charkhchi what logging level will be reasonable for this and other places where you use it? I think here, for instance, it should at least be a logging.warn.

Reply:
@jyobhai We should use logging.error for errors that we want to raise.